packages/corvid-local-testkit/README.md

```diff
@@ -1 +1,24 @@
-#
\ No newline at end of file
+# Corvid Local Testkit
+ This test kit exposes the localMode express server and drivers to the local file system
+
+### Example
+Clone Mode:
+```
+const {server, localSite} = require('corvid-local-testkit')
+
+const cloneLocalSiteDriver = await localSite.init()
+const cloneLocalServerDriver = await server.startInCloneMode(localSiteDriver)
+
+```
+Edit Mode:
+```
+const {server, localSite} = require('corvid-local-testkit')
+
+/*
+* first you need to get valid local files (clone command)
+*/
+const localFiles = await cloneLocalSiteDriver.getFiles()
```

Comment:
getFiles won't return anything...

Reply:
it's assuming that the user did a successful clone, and that cloneLocalSiteDriver is the driver of that cloned site.

Reply:
but it's under "editMode"...

Reply:
correct, because you can't do edit without performing clone before.
maybe it's not clear..
I can do something like:

**Edit Mode:**
const cloneLocalSiteDriver = await localSite.init()
const cloneLocalServerDriver = await server.startInCloneMode(cloneLocalSiteDriver)


const localFiles = await cloneLocalSiteDriver.getFiles()

const editLocalSiteDriver = await localSite.init(localFiles)
const editLocalServerDriver = await server.startInEditMode(localSiteDriver)

Reply:
make sense?

Reply:
yes, or just show an example for localFiles (even without real values)